docs/ai_actions/install_ai_actions.md

```diff
@@ -32,10 +32,72 @@ Then, in the root folder of your project, modify the `.env` file: find the `OPEN
 
 ```bash
 ###> ibexa/connector-openai ###
-OPENAI_API_KEY=sk-svcacct-AFCrCt1h2s3i4s5i6s7t8h9e0a1p2i3c4o5d6e
+OPENAI_API_KEY=<your_api_key>
 ###< ibexa/connector-openai ###
 ```
 
+### Configure access to [[= product_name_connect =]]
+
+First, get the credentials by contacting [Ibexa Support](https://support.ibexa.co).
+
+#### Create team
+
+In [[= product_name_connect =]], set up the account, and [create a team](https://doc.ibexa.co/projects/connect/en/latest/access_management/teams/#creating-teams).
+Navigate to the team details page and note down the numerical value of the **Team id** variable.
+
+Creating a team matters, because [scenarios](https://doc.ibexa.co/projects/connect/en/latest/scenarios/creating_a_scenario/) that process data coming from your AI action are associated with a team.
```

Comment:
```suggestion
Creating a team matters, because [scenarios]([[= connect_doc =]]/scenarios/creating_a_scenario/) that process data coming from your AI action are associated with a team.
```